docs/markup-variables-blocks.md

```diff
@@ -3,4 +3,184 @@ id: markup-variables-blocks
+
+This functionality provides us with the ability to bind variables into the natural language, and this concept of a variable extends further than a single value. We can insert a block of variables or even complex [Ergo logic expressions](https://docs.accordproject.org/docs/markup-expr.html),
+
+Block Formulas
+---
+
+Variables can be expanded into blocks and conditionals, such as ordered lists or `if...else` formulas. The functionality is extended from basic variables, so these block variables also create (structured) text and support the parsing and generation of the document text, and do not require Ergo knowledge.
+
+Block Formulas of kind `kind` with scope `scope` and parameters `params` are quoted with the following syntax: 
+
+```handlebars
+{{#kind scope params}}
+...
+{{/kind}}
+```
+
+### Unordered Lists
+
+
+
+```handlebars
+{{#ulist products}}
+Product {[name}} with code {[id}}
+{{/ulist}}
+```
+
+This code is rendered as markdown from a JSON object created through [Concerto](https://docs.accordproject.org/docs/model-concerto.html):
+
+```js
+{ products: [
+    { name: "Sponge", id: 1312 },
+    { name: "Bleach", id: 341 }
+  ] }
+```
+
+Which results in the following markdown text:
+
+```md
+- Product "Sponge" with code 1312
+- Product "Bleach" with code 341
+```
+
+### Ordered Lists
+
+```handlebars
+{{#olist products}}
+Product {{name}} with code {{id}}
+{{/olist}}
+```
+
+This code is rendered as markdown from a JSON object created through [Concerto](https://docs.accordproject.org/docs/model-concerto.html):
+
+```js
+{ products: [
+    { name: "Sponge", id: 1312 },
+    { name: "Bleach", id: 341 }
+  ] }
+```
+
+Which results in the following markdown text:
+
+```md
+1. Product "Sponge" with code 1312
+1. Product "Bleach" with code 341
+```
+
+### Joined Lists
+
+You create a list with a delimiter
+```handlebars
+{{#join products ","}}
+{{name}}
+{{/join}}
+```
+
+This code is rendered as markdown from a JSON object created through [Concerto](https://docs.accordproject.org/docs/model-concerto.html):
+
+```js
+{ products: [
+    { name: "Sponge", id: 1312 },
+    { name: "Bleach", id: 341 }
+  ] }
+```
+
+Which results in the following markdown text:
+
+```md
+"Sponge","Bleach"
+```
+
+### Clauses
+
+Block variable formulas support clauses to be defined inline within the markdown:
+
+```handlebars
+Payment
+-------
+{{#clause payment}}
+As consideration in full for the rights granted herein, Licensee shall pay Licensor a one-time
+fee in the amount of {{amountText}} ({{amount}}) upon execution of this Agreement, payable as
+follows: {{paymentProcedure}}.
+{{/clause}}
+```
+_Note_: There is no current support for clauses to be be imported.
```

Comment:
I would remove. We should have a GitHub issue instead (there probably is one already).